### PR TITLE
Enable incremental server-side regeneration for store pages

### PR DESCRIPTION
**Pull request by piotrgrundas**
[SALEOR-1566](https://app.clickup.com/t/2549495/SALEOR-1566)

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1566 Enable incremental server-side regeneration for store pages](https://app.clickup.com/t/2549495/SALEOR-1566)

**Review by jwm0 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-- Node.js 12.0+
+- Node.js between 12.0 & 14.0
```
[the remark this answers is not in the document]
Comment (patrys) [reply README.md:31]: What do we gain from supporting Node.js 12.x? What if we instead said, "use the latest 14.x"?

File: src/@next/components/molecules/OrderTable/OrderTable.tsx [hunk 2]
```diff
@@ -52,61 +52,59 @@ export const OrderTable: React.FC<IProps> = ({ orders, isGuest }: IProps) => {
...
+                      href={generatePath(
+                        isGuest
                           ? paths.guestOrderDetail
                           : paths.accountOrderDetail,
-                        query: { token: order.node.token },
-                      }}
-                      key={order.node.number}
+                        { token }
+                      )}
+                      key={number!}
```
[the remark this answers is not in the document]
Comment (piotrgrundas) [reply src/@next/components/molecules/OrderTable/OrderTable.tsx:66]: I don't think that ever order without number may be created by a customer.

File: package.json [hunk 3]
```diff
@@ -7,33 +7,29 @@
...
-    "@babel/runtime": "^7.5.5",
-    "@editorjs/editorjs": "^2.19.0",
+    "@editorjs/editorjs": "^2.19.1",
     "@editorjs/header": "^2.6.0",
     "@editorjs/image": "^2.6.0",
     "@editorjs/list": "^1.6.0",
     "@editorjs/quote": "^2.4.0",
-    "@lhci/cli": "^0.6.1",
+    "@lhci/cli": "^0.7.0",
+    "@saleor/sdk": "github:mirumee/saleor-sdk#update_storefront_types",
```
Comment (orzechdev) [inline package.json:16]: We should release a new version of sdk and use it here before merging PR to master

File: src/@next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx [hunk 4]
```diff
@@ -14,26 +14,26 @@ export const AttributeValuesChecklist: React.FC<IProps> = ({
...
       {title && <S.Header>{title}</S.Header>}
       {values &&
-        values.map((value, index) => {
-          if (!viewAllOptions && index > valuesShowLimitNumber - 1) {
-            return <></>;
-          }
-          return (
+        values.map((value, index) =>
+          !viewAllOptions && index > valuesShowLimitNumber - 1 ? (
+            <Fragment key={index} />
```
Comment (orzechdev) [inline src/@next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx:25]: Why is the empty fragment used here at all? Can't we remove this whole condition?

File: package.json (hunk 3 above)
Comment (piotrgrundas) [reply package.json]: Obviously

File: src/@next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx (hunk 4 above)
Comment (piotrgrundas) [reply src/@next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx:25]: U can ask the author.. [which is you](https://github.com/mirumee/saleor-storefront/blame/master/src/%40next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx#L25) :)
I've just fixed console error.

**Review by jwm0 — APPROVED** [review]

**Pull request merged**